modules/local/gffread.nf

```diff
@@ -0,0 +1,50 @@
+process GFFREAD {
+    tag "$gff"
+    label 'process_low'
+
+    conda "bioconda::gffread=0.12.1"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gffread:0.12.1--h8b12597_0' :
+        'biocontainers/gffread:0.12.1--h8b12597_0' }"
+
+    input:
+    tuple val(meta), path(gff)
+    tuple val(meta), path(fasta)
+
+    output:
+    path "*.gtf"        , emit: gtf,        optional: true
+    path "*.fa"         , emit: tr_fasta,   optional: true
```

Comment:
I think these glob pattern might apply to input files as well. I would see if it is possible to change it to something more specific.

Reply:
also you can include a check for this case as done 
[here](https://github.com/nf-core/modules/blob/39656f68219340420f03bd54a68e111c86e107e6/modules/nf-core/bcftools/filter/main.nf#L30)